config/config.md

```diff
@@ -0,0 +1,166 @@
+    // above.
+    "atnaUrl": "mongodb://localhost/openhim"
+  },
+  "logger": {
+    // the logging level
+    "level": "info"
+  },
+  "router": {
+    // This is the port that the OpenHIM will recieve HTTP request on to
+    // them to other services
+    "httpPort": 5001,
+    // This is the port that the OpenHIM will recieve HTTPS request on to
+    // them to other services
+    "httpsPort": 5000,
+    // The timout for requests that the OpenHIM makes to other services
+    "timeout": 60000
+  },
+  "api": {
+    // The port that the OpenHIM API uses
+    "httpsPort": 8080,
+    // API request are only valid for a particular window once made, this is
+    // the size of that window in seconds
+    "authWindowSeconds": 10
+  },
+  "rerun": {
+    // The port that the transaction re-run processor runs on, this port is
+    // used internally and SHOULD NOT be expposed past your firewall
+    "httpPort": 7786,
+    // The host where the re-run processor is running (always the OpenHIM
+    // server)
+    "host": "localhost",
+    "processor": {
+      // if the processor is enabled or not
+      "enabled": true,
+      // The period to poll for new re-run tasks to run
+      "pollPeriodMillis": 2000
+    }
+  },
+  "tcpAdapter": {
+    "httpReceiver": {
+      // the port that the HTTP reciever for TCP transaction runs on. This
+      // is an internally used port and SHOULD NOT be expposed past your
+      // firewall.
+      "httpPort": 7787,
+      // The host where the tcp adpater's http reciever is running (always
+      // the OpenHIM server)
+      "host": "localhost"
+    }
+  },
+  "polling": {
+    // the port used by the polling channel processor, this is an internally
+    // used port and SHOULD NOT be exposed past your firewall.
+    "pollingPort": 7788,
+    // The host where the polling processor is running (always the OpenHIM
+    // server)
+    "host": "localhost"
+  },
+  "authentication": {
+    // What kid of authentication methods to allow
+    "enableMutualTLSAuthentication": true,
+    "enableBasicAuthentication": true
+  },
+  "nodemailer": {
+    // The nodemailer service to use see: https://github.com/andris9/Nodemailer
+    "service": "Gmail",
+    "auth": {
+      "user": "user@gmail.com",
+      "pass": "password"
+    }
+  },
+  "smsGateway": {
+    "provider": "clickatell",
+    "config": {
+      "user": "user",
+      "pass": "pass",
+      "apiID": "apiID"
+    }
+  },
+  "alerts": {
+    // To enable alerting when things go bad
+    "enableAlerts": true,
+    // How often to poll for alerts
+    "pollPeriodMinutes": 1,
+    // The name of the OpenHIM instance to use in the alert and report text
+    "himInstance": "openhim.jembi.org",
+    // The URL of the OpenHIM instance to use for links in the alert and
+    // report text
+    "consoleURL": "http://openhim.jembi.org"
+  },
+  "reports": {
+    // To enable daily and weekly reporting
+    "enableReports": true
+  },
+  "visualizer": {
```

Comment:
I can try help expand this a bit?

```js
// The minimum duration to use for the event.
// If the event's duration is less than this specified period,
// then the end time will be adjusted in order to match
"minimumEventPeriodMillis": 100,
// Should orchestration and route timestamps be normalized to the transaction time
// This may be useful if you have mediators running on other servers
"enableTSNormalization": true,
// A buffer period for orchestration timestamps
"orchestrationTSBufferMillis": 100
```